internal/services/hdinsight/hdinsight_spark_cluster_resource.go

```diff
@@ -150,6 +150,8 @@ func resourceHDInsightSparkCluster() *pluginsdk.Resource {
 			"monitor": SchemaHDInsightsMonitor(),
 
 			"extension": SchemaHDInsightsExtension(),
+
+			"availability_zones": commonschema.ZonesMultipleOptional(),
```

Comment:
Convention in the provider is to call these `zones`
```suggestion
			"zones": commonschema.ZonesMultipleOptional(),
```

Reply:
Changed.